test/test.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+import json
+
+import diff
+
+
+# diff.compare_objects(obj1, obj2) returns true if equall and false if different coverage
+def test_compare_objects_same_coverage_same_functions():
+    obj1 = json.loads('{"functions": [{"name": "func1", "start": 36, "exec": true}, {"name": "func2", "start": 64, "exec": false}], "branches": [], "name": "obj-firefox/dist/include/mozilla/dom/DOMRectListBinding.h", "coverage": [null, 2, null, null, 5], "source_digest": "another_generated_number_with_literals"}')
+    obj2 = json.loads('{"functions": [{"name": "func1", "start": 36, "exec": true}, {"name": "func2", "start": 64, "exec": false}], "branches": [], "name": "obj-firefox/dist/include/mozilla/dom/DOMRectListBinding.h", "coverage": [null, 2, null, null, 5], "source_digest": "another_generated_number_with_literals2"}')
+    assert diff.compare_objects(obj1, obj2) is None
+
+
+def test_compare_objects_same_coverage_diff_functions():
+    obj1 = json.loads('{"functions": [{"name": "func1", "start": 36, "exec": true}, {"name": "func2", "start": 64, "exec": true}], "branches": [], "name": "obj-firefox/dist/include/mozilla/dom/DOMRectListBinding.h", "coverage": [null, 2, null, null, 5], "source_digest": "another_generated_number_with_literals"}')
+    obj2 = json.loads('{"functions": [{"name": "func1", "start": 36, "exec": true}, {"name": "func2", "start": 64, "exec": false}], "branches": [], "name": "obj-firefox/dist/include/mozilla/dom/DOMRectListBinding.h", "coverage": [null, 2, null, null, 5], "source_digest": "another_generated_number_with_literals2"}')
+    assert json.loads(diff.compare_objects(obj1, obj2)) == {'branches': [], 'functions': [], 'coverage': [None, 0, None, None, 0], 'source_digest': 'another_generated_number_with_literals', 'name': 'obj-firefox/dist/include/mozilla/dom/DOMRectListBinding.h'}
+
+
+def test_compare_objects_diff_coverage_same_func():
```

Comment:
We should test more cases for diff coverage.
E.g. a case where a line is covered in A but not covered in B.